src/javascripts/ng-admin/Crud/list/maDatagridPagination.js

```diff
@@ -18,33 +17,11 @@ define(function (require) {
             },
             template: paginationView,
             controllerAs: 'paginationCtrl',
-            controller: DatagridPaginationController,
-            link: function (scope, element, attrs, controller) {
```

Comment:
As it didn't make sense to have code both in the `link` function and in the controller, I moved this part in `DatagridPaginationController`.